libs/base/System.idr

```diff
@@ -62,7 +64,7 @@ time = do MkRaw t <- foreign FFI_C "idris_time" (IO (Raw Integer))
           pure t
 
 ||| Specify interval to sleep for, must be in range [0, 1000000]
```

Comment:
Sorry small comment again, but it would be nice if you could write the range as `]0, 1000000]` to show that it is exclusive.